legal-api/migrations/versions/957a67bde783_add_in_liquidation_column_to_businesses.py

```diff
@@ -0,0 +1,29 @@
+"""Add in_liquidation column to businesses
+
+Revision ID: 957a67bde783
+Revises: 1ec937c2466e
+Create Date: 2025-05-29 11:43:23.869675
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '957a67bde783'
+down_revision = '1ec937c2466e'
```

Comment:
A PR merged with a new revision (ref: https://github.com/bcgov/lear/pull/3536/files#diff-0e5e8f1c31ba60d958137a4bebd146c630fb172e50cf3188e7033d25a66c58f6R14)
```suggestion
down_revision = '0c3eb026efa0'
```